cmake/CMakeLists.txt

```diff
@@ -1242,6 +1243,10 @@ if (onnxruntime_USE_DML)
   include(dml)
 endif()
 
+if (onnxruntime_ENABLE_TRAINING_OPS)
+  add_compile_definitions(ENABLE_TRAINING_OPS)
+endif()
+
 if (onnxruntime_ENABLE_TRAINING)
   add_compile_definitions(ENABLE_TRAINING)
 
```

Comment:
should we add add_compile_definitions(ENABLE_TRAINING_OPS) here to enforce 
enable_training is a strict superset of enable_training_ops.

Reply:
I added it, but now that I look at it again I may have put it in a goofy place. Let me know if you want me to move it.